share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
Building from source does not get you higher security, it gets you lower. We do not use a sandbox so you are executing arbitrary buildsystem code on your machine as an admin user. Building from source is significantly less reliable and significantly slower as it relies heavily on the user's external environment. If you look at our issues list: they are almost all building from source issues.

Look, I'm not saying we remove `--build-from-source` as a flag. I'm saying telling Homebrew "I never want your binary packages" is not something we should support or document. I've just come back from a week long meeting with maintainers from Debian, MacPorts, FreeBSD, NetBSD and various other Linux distros. Many of them were amazed why we allow the amount of options we do (e.g. build from source, pick your compiler, pick your prefix, optional dependencies, install from HEAD). The more options and entropy: the worse the user experience for the majority of users in favour of a small, vocal minority. That is a bad way of doing software. Obviously anyone is welcome to help out and fix it but I don't think any individuals have enough time to fix this and this is why most software is distributed as in binary form.